### ISSUE TITLE
An ADA isue: the typeahead suggestions in the search field of the website's header are not announced by the screen reader. Hence, the screen reader user won’t be made aware of the suggestions based on entered keyword.

### ISSUE THREAD

**Issue by dmitrii-fediuk** [kitchencollection-com/core#34]
[upwork.com/ab/f/contracts/20874283](https://www.upwork.com/ab/f/contracts/20874283)

![03](https://user-images.githubusercontent.com/83204/47614041-7cad2500-dac3-11e8-96f7-0d35f844f28f.png)

## Rationale
> Autocomplete widgets can be helpful for accessibility because they can make it easier to enter
text by providing suggestions based on the characters initially typed.
This particularly helps people who find typing more difficult and people who may be susceptible to spelling mistakes.

## Recommendations
> Use an `aria-live` region to announce when autocomplete options are available, and to create
a listbox containing the options that appear as the user types, which is controlled with the up and
down arrow keys.

**Comment by dmitrii-fediuk** [comment]
Done.
